lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -78,6 +78,14 @@ def date_range(self, start: AnyStr, end: AnyStr) -> list:
             date_range.append(f"{m.year:04}-{m.month:02}")
         return date_range
 
+    def test_malformed_authorization_header(self, client, server_config):
```

Comment:
Thats a good idea but probably I will have to do in both file `commons.py` and `test_datasets_publish.py` since that overrides the `test_malformed_authorization_header` method